roles/openshift_node_upgrade/tasks/restart.yml

```diff
@@ -22,7 +22,6 @@
     - "{{ openshift.common.service_type }}-master-controllers"
     - "{{ openshift.common.service_type }}-node"
   failed_when: false
-  when: openshift.common.is_containerized | bool
```

Comment:
In the non-containerized deployment master services are never stopped. However, given the `systemctl start` over running services has no effect, it is completely fine to remove the when condition.